### ISSUE TITLE
Steps get greyed out after preview

### ISSUE THREAD

**Issue by PapitoPapito** [brunomikoski/Animation-Sequencer#81]
When I play the preview, all steps parameter get greyed out. I have to select another gameobject in the hierarchy and reselect the Animation sequencer to be able to tweak again the parameters which made testing a lot more slower. Is it possible to avoid that?

**Comment by snappedToGrid** [comment]
Yea I get the same issue. Did you mange to solve this?

**Comment by brunomikoski** [comment]
Hi! Sorry about the delay on checking this issue, what version of unity you are using it?

**Comment by snappedToGrid** [comment]
Thanks!
I'm using v2022.3.38f1

I hope this helps.
Thanks again.

**Comment by brunomikoski** [comment]
Just tested here on Unity 2022.3.16f1
And DOTWeen 1.2.765

And everything is looking alright. Are you tryig to use on Prefab Isolation? Dotween editor simulation doesn't work really well with it. 

https://github.com/user-attachments/assets/522458b6-8f2c-403a-9e77-5df393d6ab37

**Comment by snappedToGrid** [comment]
Oh! I think when clicking the stop button it works just fine in isolation mode as well, I think the "issue" was that I didn't think it was necessary to click the stop button to make changes. Maybe a clarification could be helpful? Clicking on the stop button to make changes is definitely not a problem once you know it's needed.

Thanks for the video and the help, it made it really easy to find out what the source of the issue has been.

**Comment by brunomikoski** [comment]
Yeah, sorry this is one of the gimmicks of the DOTween! I will add some clarification for further versions.
